event_manager/lib/validate_phone_number.rb

```diff
@@ -1,12 +1,18 @@
 class ValidatePhoneNumber
   def call(number)
```

Comment:
I would rather use method name `validate` and class method, i.e. `self.validate` name here.